### ISSUE TITLE
App may show wrong images after an app update if you add new images

### ISSUE THREAD

**Issue by tiago-yoneda** [facebook/fresco#2664]
### Description
Using version 2.6.0
We added several png assets in our app. Then after upgrading to the new version (not a fresh install) , some images showed up in the wrong place.

The thing is it does not happen 100% of the times we added an asset. But it did happen several times through our builds.

### Solution
Downgrading to version 2.5.0 fixed it.
We did not test other versions.

**Comment by pierroo** [comment]
Hi @tiago-yoneda , I am also facing the same issue.
It is completely random, only on production build, and not for every users.

How do you downgrade to version  2.5.0?

I read online I should add the following lines before the dependencies in app/build.gradle:

configurations.all {
    resolutionStrategy {
        force "com.facebook.fresco:fresco:2.5.0"
    }
}

Is this how you do it?
The line above is enough?
or do we have to add the same lines for every fresco 2.6.0 (like screenshot above)

![image](https://user-images.githubusercontent.com/17619412/197334000-39aa53f8-dfda-48e2-aff5-c7bc298ea555.png)


Thank you so much for your help if you happen to pass by :)

**Comment by tiago-yoneda** [comment]
@pierroo I noticed they answered your question in that issue (#2667) thread. Hope it works for you.

**Comment by pierroo** [comment]
Yes indeed, I had to downgrade every single mention of fresco to 2.5.0 and eventually it worked, thank you :)

**Comment by tiago-yoneda** [comment]
@pierroo , it seems that a version 3.0.0 has been released, but I've yet to test it. However the the issue above #2667 they left a message saying our problem was fixed. 

Let's hope that's the case.
